app/services/green_lanes/determine_next_page.rb

```diff
@@ -31,6 +31,7 @@ def single_category?
     end
 
     def handle_cat1_cat2(cat_1_exemptions_apply)
+      return check_your_answers if @determine_category.cat1_without_exemptions.present?
```

Comment:
If any cat 1 assessments dont have exemptions then they have to go straight to the CYA page as they cant be exemption from them